src/js/cf-expandables.js

```diff
@@ -5,110 +5,255 @@
+                 );
+    },
 
-  $.fn.expandable.calculateExpandDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 450, 900, height * 4 );
-  };
+    /**
+     * Expands or collapses the Expandable.
+     *
+     * @param {object} event - jQuery event object.
+     */
+    toggle: function toggle( event ) {
+      event.preventDefault();
+      event.stopPropagation();
 
-  $.fn.expandable.calculateCollapseDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 350, 900, height * 2 );
-  };
+      if ( this.isInAccordion ) {
+        this.$el.siblings( '.expandable' ).each( function( index, sibling ) {
+          sibling.collapse();
+        } );
+      }
+
+      if ( this.isExpanded ) {
+        this.collapse();
+      } else {
+        this.expand();
+      }
+    },
 
-  $.fn.expandable.constrainValue = function( min, max, duration ) {
-    if ( duration > max ) {
-        return max;
-    } else if ( duration < min ) {
-        return min;
-    } else {
-        return duration;
+    /**
+     * Expands the expandable, sets the internal state
+     * and adds aria attributes.
+     *
+     * @param {integer} duration -
+     * The time duration in which the expansion will occur.
+     */
+    expand: function expand( duration ) {
+      var $content = this.$content;
+      this.$target.attr( 'aria-pressed', 'true' );
+      $content.attr( 'aria-expanded', 'true' );
+
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+
+      this.$el.addClass( this.expandedClass );
+
+      $content.slideDown( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+
+      this.isExpanded = true;
+    },
+
+    /**
+     * Collaspses the expandable, sets the internal state
+     * and adds aria attributes.
+     *
+     * @param {integer} duration -
+     * The time duration in which the collaspe will occur.
+     */
+    collapse: function collapse( duration ) {
+      var $content = this.$content;
+      this.$target.attr( 'aria-pressed', 'false' );
+      $content.attr( 'aria-expanded', 'false' );
+
+      if ( typeof duration === 'undefined' ) {
+        duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      }
+
+      this.$el.removeClass( this.expandedClass );
+
+      $content.slideUp( {
+        duration: duration,
+        easing:   'easeOutExpo'
+      } );
+
+      this.isExpanded = false;
     }
+
+  };
+
+
+  // Expandable Plugin declarations.
+
+
+  /**
+   * Instantiates the Expandable and configures
+   * the expandable options.
+   *
+   * @param {object} options - Object used to customize the Expandable.
+   * @returns {object} - jQuery Object.
+   */
+  $.fn.expandable = function expandable( options ) {
+
+    return this.each( function() {
+      ( options || ( options = {} ) ).$el = $( this );
```

Comment:
Nope, need the extra set of parens to evaluate the assignment statement as expression.